source/components/cardContainer/card/headerColumn/sizeForBreakpoints.tests.ts

```diff
@@ -0,0 +1,74 @@
+import { SizeForBreakpoints } from './sizeForBreakpoints';
+
+import { services } from 'typescript-angular-utilities';
+import __string = services.string;
+import __object = services.object;
+
+describe('SizeForBreakpoints', () => {
+	let sizeForBreakpoints: SizeForBreakpoints;
+
+	beforeEach(() => {
+		sizeForBreakpoints = new SizeForBreakpoints(__string.stringUtility, __object.objectUtility);
+	});
+
+	it('should convert an object of breakpoint sizes into a class string', () => {
+		const sizes = {
+			lg: 4,
+			md: 3,
+			sm: 2,
+			xs: 1,
+		};
+		expect(sizeForBreakpoints.getClass(sizes, '')).to.equal('col-xs-1 col-sm-2 col-md-3 col-lg-4 ');
```

Comment:
That's a good suggestion, but I think these are certainly still useful. Part of the purpose of unit tests is that they enforce that when you go and make a code change, the behavior still lines up with what it did before, or changes in a way that you intended (by updating the tests). Ideally, if you change the css framework you'd first update these tests to show what the classes should look like, then go update the code to match.